Tests/TestHelpers.swift

```diff
@@ -101,17 +102,19 @@ extension GitHubUserContent: TargetType {
         switch self {
         case .downloadMoyaWebContent(let contentPath):
             return "/Moya/Moya/master/web/\(contentPath)"
+        case .requestMoyaWebContent(let contentPath):
+            return "/Moya/Moya/master/web/\(contentPath)"
         }
```

Comment:
Can these two cases be merged into one since they have the same implementation? One less String 😅